src/github.com/couchbase/sync_gateway/db/event.go

```diff
@@ -143,9 +144,13 @@ func (ef *JSEventFunction) CallFunction(event Event) (interface{}, error) {
 	switch event := event.(type) {
 
 	case *DocumentChangeEvent:
-		result, err = ef.Call(event.Doc)
+		if (len(event.OldDoc) == 0) {
+			result, err = ef.Call(event.Doc, nil)
```

Comment:
Do we need to pass nil here?  I think Call takes a variable number of args.  Same thing below for the DBStateChangeEvent.